pages/index.tsx

```diff
@@ -57,29 +64,18 @@ export default function Home() {
 
 	const [products, setProducts] = React.useState<Product[]>([]);
```

Comment:
Can you remove products and setProducts? allProducts already contains all the needed products.